### PR TITLE
Added new `is_hexdigit` service on `Char`

### PR DESCRIPTION
**Pull request by R4PaSs**
Simple addition to the services of `Char`, might be useful

### CODE REVIEW

File: lib/core/text/abstract_text.nit [hunk 1]
```diff
@@ -1709,6 +1709,16 @@ redef class Char
 		return (self >= 'a' and self <= 'z') or (self >= 'A' and self <= 'Z')
 	end
 
+	# Is `self` an hexadecimal digit ?
+	#
+	#     assert 'A'.is_hexdigit
+	#     assert not 'G'.is_hexdigit
+	#     assert 'a'.is_hexdigit
+	#     assert not 'f'.is_hexdigit
```
Comment (privat) [inline lib/core/text/abstract_text.nit]: `'f'` should be in fact, maybe you want `'g'` ?

Comment (R4PaSs) [inline lib/core/text/abstract_text.nit]: vfgwh

**Comment by xymus** [comment]
+1

**Comment by privat** [comment]
+1

**Pull request merged**